testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java

```diff
@@ -183,14 +183,14 @@ public void specialJVM(TestInfo testInfo) throws IOException, InterruptedExcepti
     }
 
     @Test
-    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @DisabledOnOs(value = {OS.WINDOWS}, disabledReason = "https://github.com/quarkusio/quarkus/issues/42248")
```

Comment:
Why this should be disabled only on Windows? The new issue is OS indipendent so I would be expecting that this either work on all platform or nowhere.

Reply:
Gosh, the right issue is https://github.com/quarkusio/quarkus/issues/42942